packages/inferno/src/DOM/utils/common.ts

```diff
@@ -206,7 +206,6 @@ export function moveVNodeDOM(parentVNode, vNode, parentDOM, nextNode, animations
 
     if (flags & VNodeFlags.ComponentClass) {
       refOrInstance = vNode.children;
-      instanceProps = vNode.props;
```

Comment:
Hmm? Why was this line removed?

Reply:
We shouldn't pass instant props to a Component Class since we don't do it in the other hooks (correct me if I am wrong), so I wanted to remove this inconsistency because it could cause confusion. See:

```
  componentDidMount?(): void;

  componentWillMount?(): void;
```

Reply:
Whereas I agree that breaking changes are bad, I think it is better to clean up the API early. We still have very few downloads of v8 and changing from `props` to `this.props` is intuitive. However, just updating the type definition and leaving this row would nudge people in the right direction by only throwing a type error. Then we could remove it in v9 (if we remember).